### PR TITLE
update Education-team

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

**Comment by RCheesley** [comment]
Looks good but can we use link macros rather than inline links? Otherwise a great improvement :)

File: pages/09.education-team/chapter.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ The Education Team is responsible for the Forum, Mautic Documentation and Intern
   
   * #### Sources
   
-    * [User Guide](https://docs.mautic.org/en) Describes key concepts of Mautic and provides instructions for using and contributing to Mautic.
-    * [Knowledgebase](https://kb.mautic.org) Provides a database of tutorials, FAQs, and how-to articles.
-    * [Developer Guide](https://developer.mautic.org/) details about the Mautic API, Webhooks, themes and plugin development.
-    * [Community Handbook](https://contribute.mautic.org/) central point of call for how the Mautic community is organised and managed.
+    * [User Guide][User Guide] Describes key concepts of Mautic and provides instructions for using and contributing to Mautic.
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:49]:
```suggestion
    * [User Documentation][User Documentation] Describes key concepts of Mautic and provides instructions for using and contributing to Mautic.
```

File: pages/09.education-team/chapter.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ The Education Team is responsible for the Forum, Mautic Documentation and Intern
...
+    * [Knowledgebase][Knowledgebase]Provides a database of tutorials, FAQs, and how-to articles.
+    * [Developer Guide][Developer Guide]details about the Mautic API, Webhooks, themes and plugin development.
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:51]:
```suggestion
    * [Developer Documentation][Developer Documentation] details about the Mautic API, Webhooks, themes and plugin development.
```

File: pages/09.education-team/chapter.md [hunk 3]
```diff
@@ -77,13 +77,13 @@ The Education Team is responsible for the Forum, Mautic Documentation and Intern
     
    * #### Style Guide
    
-     * [The Mautic Style Guide](https://docs.mautic.org/en/contributing/style-guide) recommends best practices that aims to bring about a consistent style, voice, and tone across all documentation
+     * [The Mautic Style Guide][The Mautic Style Guide]recommends best practices that aims to bring about a consistent style, voice, and tone across all documentation
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:80]:
```suggestion
     * [The Mautic Style Guide][The Mautic Style Guide] recommends best practices that aims to bring about a consistent style, voice, and tone across all documentation
```

File: pages/09.education-team/chapter.md [hunk 4]
```diff
@@ -115,4 +115,14 @@ We need people with skills as:
 
 
 
-** Would you like to be involved in this team? Join [#t-education](https://mautic.slack.com/archives/CQGQ0D4KU) on Slack! ** 
+** Would you like to be involved in this team? Join [#t-education][#t-education] on Slack! ** 
+
+
+
+[User Guide]:<https://docs.mautic.org/en>
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:122]:
```suggestion
[User Documentation]: <https://docs.mautic.org/en>
```

File: pages/09.education-team/chapter.md [hunk 5]
```diff
@@ -115,4 +115,14 @@ We need people with skills as:
...
+[Knowledgebase]:<https://kb.mautic.org> 
+[Developer Guide]:<https://developer.mautic.org/>
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:124]:
```suggestion
[Developer Documentation]: <https://developer.mautic.org/>
```

File: pages/09.education-team/chapter.md [hunk 6]
```diff
@@ -115,4 +115,14 @@ We need people with skills as:
 
 
 
-** Would you like to be involved in this team? Join [#t-education](https://mautic.slack.com/archives/CQGQ0D4KU) on Slack! ** 
+** Would you like to be involved in this team? Join [#t-education][#t-education] on Slack! ** 
+
+
+
+[User Guide]:<https://docs.mautic.org/en>
+[Knowledgebase]:<https://kb.mautic.org>
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:123]:
```suggestion
[Knowledgebase]: <https://kb.mautic.org> 
```

File: pages/09.education-team/chapter.md [hunk 7]
```diff
@@ -115,4 +115,14 @@ We need people with skills as:
...
 
-** Would you like to be involved in this team? Join [#t-education](https://mautic.slack.com/archives/CQGQ0D4KU) on Slack! ** 
+** Would you like to be involved in this team? Join [#t-education][#t-education] on Slack! ** 
+
+
+
+[User Guide]:<https://docs.mautic.org/en>
+[Knowledgebase]:<https://kb.mautic.org> 
+[Developer Guide]:<https://developer.mautic.org/>
+[Community Handbook]:<https://contribute.mautic.org/>
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:125]:
```suggestion
[Community Handbook]: <https://contribute.mautic.org/> 
```

File: pages/09.education-team/chapter.md [hunk 8]
```diff
@@ -115,4 +115,14 @@ We need people with skills as:
...
-** Would you like to be involved in this team? Join [#t-education](https://mautic.slack.com/archives/CQGQ0D4KU) on Slack! ** 
+** Would you like to be involved in this team? Join [#t-education][#t-education] on Slack! ** 
+
+
+
+[User Guide]:<https://docs.mautic.org/en>
+[Knowledgebase]:<https://kb.mautic.org> 
+[Developer Guide]:<https://developer.mautic.org/>
+[Community Handbook]:<https://contribute.mautic.org/> 
+[Mautic Community Forums]:<https://forum.mautic.org>
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:126]:
```suggestion
[Mautic Community Forums]: <https://forum.mautic.org>
```

File: pages/09.education-team/chapter.md [hunk 9]
```diff
@@ -115,4 +115,14 @@ We need people with skills as:
...
+[The Mautic Style Guide]:<https://docs.mautic.org/en/contributing/style-guide>
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:127]:
```suggestion
[The Mautic Style Guide]: <https://docs.mautic.org/en/contributing/style-guide>
```

File: pages/09.education-team/chapter.md [hunk 10]
```diff
@@ -115,4 +115,14 @@ We need people with skills as:
...
+[#t-education]:<https://mautic.slack.com/archives/CQGQ0D4KU>
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:128]:
```suggestion
[#t-education]: <https://mautic.slack.com/archives/CQGQ0D4KU>
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Just a couple of missing spaces to fix, also I think that we should refer to the user/dev docs as they are known (user documentation/developer documentation) rather than guide as it may confuse some folk?  See what you think :)

**Comment by fakela** [comment]
All changes resolved

**Review by RCheesley — APPROVED** [review]

**Pull request merged**